qt/NodeunitTest/TestEnvironment.qml

```diff
@@ -0,0 +1,42 @@
+import QtQuick 2.0
+import FS 1.0 as FS
+import "../../js/lib/ilib-qt.js" as QtIlib
+import "../../js/test/nodeunit/nodeunit-qml.js" as Nodeunit
+
+QtObject {
+	id: thisObj
+	property string path: ""
+    property string moduleName: ""
+    property var ilib: {}
+    property var require: {}
+	
+    Component.onCompleted: {
+        //console.log(">>>>>>>>>>>> [TestEnvironment.qml] new context. Loading in a fresh copy of ilib.");
+        console.log("[TestEnvironment.qml] (path) : " + path + " (moduleName): " + moduleName);
+
+        ilib = QtIlib.ilib;
+        var loader = new QtIlib.QmlLoader(FS.FileReader);
+        ilib.setLoaderCallback(loader);
+		require = QtIlib.require;
+
+        var testSuites, runTest, i;
+        testSuites = require("qmltest", path);
+
+        if (moduleName === "name") {
+            for (i=0; i < testSuites["namefmt"].length; i++) {
```

Comment:
normally moduleName is same with test directory name.  But, when the module name is "name" and "string-ext". It couldn't require file properly.  I assume "string-ext" is because of "-" and "name" is kind of 'keyword' that should not be used.  so I had to give module name differently